src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+     * @param args System Arguments added to program.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        /**
+         * ASCII art source: https://www.twitchquotes.com/copypastas/2059
+         */
+        String sadge = "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠿⠛⠋⠉⣉⣉⠙⠿⠋⣠⢴⣊⣙⢿⣿⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠟⠋⠁⠀⢀⠔⡩⠔⠒⠛⠧⣾⠊⢁⣀⣀⣀⡙⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⠟⠛⠁⠀⠀⠀⠀⠀⡡⠊⠀⠀⣀⣠⣤⣌⣾⣿⠏⠀⡈⢿⡜\n"
+                + "   ⣿⣿⣿⠿⠛⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠡⣤⣶⠏⢁⠈⢻⡏⠙⠛⠀⣀⣁⣤⢢\n"
+                + "   ⣿⠋⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠰⣄⡀⠣⣌⡙⠀⣘⡁⠜⠈⠑⢮⡭⠴⠚⠉⠀\n"
+                + "   ⠁⠀⢀⠔⠁⣀⣤⣤⣤⣤⣤⣄⣀⠀⠉⠉⠉⠉⠉⠁⠀⠀⠀⠀⠀⠁⠀⢀⣠⢠\n"
+                + "   ⡀⠀⢸⠀⢼⣿⣿⣶⣭⣭⣭⣟⣛⣛⡿⠷⠶⠶⢶⣶⣤⣤⣤⣶⣶⣾⡿⠿⣫⣾\n"
+                + "   ⠇⠀⠀⠀⠈⠉⠉⠉⠉⠉⠙⠛⠛⠻⠿⠿⠿⠷⣶⣶⣶⣶⣶⣶⣶⣶⡾⢗⣿⣿\n"
+                + "   ⣦⣄⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣴⣿⣶⣾⣿⣿⣿\n"
+                + "   ⣿⣿⣿⣷⣶⣤⣄⣀⣀⣀⡀⠀⠀⠀⠀⠀⠀⢀⣀⣤⣝⡻⣿⣿⣿⣿⣿⣿⣿⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣦⡹⣿⣿⣿⣿⣿⣿\n";
+
+        /**
+         * ASCII art source: https://fsymbols.com/text-art/
+         */
+        String text = "█▀ ▄▀█ █▀▄ █▀▀ █▀▀   █▀▄ █░█ █▄▀ █▀▀\n"
+                + "▄█ █▀█ █▄▀ █▄█ ██▄   █▄▀ █▄█ █░█ ██▄";
+
+        System.out.println("Hello from\n" + sadge + text);
+
+        System.out.println(LINES);
+        System.out.println(GREETING);
+        System.out.println(LINES);
+
+        Scanner in = new Scanner(System.in);
+        while (!done) {
+            String input = in.nextLine();
+            parseInput(input);
+        }
+
+
+    }
+
+    /**
+     * This function parses the input given by user and prints the appropriate response.
+     *
+     * @param input input given by the user.
+     */
+    private static void parseInput(String input) {
+        System.out.println(LINES);
+
+        if (input.equalsIgnoreCase("Bye")) {
+            System.out.println(GOODBYE);
+            setDone();
+        } else if (input.equalsIgnoreCase("List")) {
+            printList();
+        } else if (isDoneInput(input)) {
+            int index = getDoneIndex(input);
+            list[index].markAsDone();
+            System.out.print(TAB + "Nice! I've marked this task as done:" + System.lineSeparator() + TAB + TAB);
+            System.out.println(list[index]);
+        } else if (isValidTodo(input)) {
+            addToList(input, TODO);
+            printAddedMessage();
+        } else if (isValidDeadline(input)) {
+            addToList(input, DEADLINE);
+            printAddedMessage();
+        } else if (isValidEvent(input)) {
+            addToList(input, EVENT);
+            printAddedMessage();
+        } else {
+            setError(true);
+        }
+        if (error) {
+            System.out.println(TAB + "Invalid input");
+            setError(false);
+        }
+        System.out.println(LINES);
+    }
```

Comment:
Your method is almost hitting >30 LOCs, try splitting them into different methods?
https://nus-cs2113-ay2122s1.github.io/website/schedule/week4/topics.html#implementation-code-quality-readability-basic-avoid-long-methods